apps/nextjs-website/src/_contents/appIo/overview.ts

```diff
@@ -112,14 +112,14 @@ export const appIoOverview: OverviewData = {
         title: 'Quick Start',
         text: 'Aderire a IO tramite l’Area Riservata, creare un servizio, verificare l’esistenza di un utente, inviare un messaggio: ecco come si fa',
         href: `${appIoQuickStartGuidePath.path}`,
-        iconName: 'FlagOutlined',
+        iconName: '/icons/flagOutline.svg',
```

Comment:
You should use Code iconName
```suggestion
        iconName: 'Code',
```